project/BuildPlugin.scala

```diff
@@ -196,7 +196,7 @@ object BuildImplementation {
   import ch.epfl.scala.sbt.release.ReleaseEarlyPlugin.{autoImport => ReleaseEarlyKeys}
 
   final val globalSettings: Seq[Def.Setting[_]] = Seq(
-    BuildKeys.schemaVersion := "1.2",
+    BuildKeys.schemaVersion := "1.2-refresh",
```

Comment:
Is that necessary ? We shouldn't need to re generate the integration configs in this PR right?

Reply:
Yes, it's necessary